src/template.json

```diff
@@ -0,0 +1,61 @@
+{
+  "builders": [
+    {
+      "ami_name": "{{user `ImageName`}}",
+      "ami_users": "{{user `AmiUsers`}}",
+      "force_delete_snapshot": "true",
+      "instance_type": "{{user `InstanceType`}}",
+      "launch_block_device_mappings": [
+        {
+          "delete_on_termination": true,
+          "device_name": "/dev/xvda",
+          "volume_size": "{{user `VolumeSize`}}",
+          "volume_type": "gp2"
+        }
+      ],
+      "profile": "{{user `AwsProfile`}}",
+      "region": "{{user `AwsRegion`}}",
+      "snapshot_users": "{{user `SnapshotUsers`}}",
+      "source_ami": "{{user `SourceImage`}}",
+      "ssh_username": "{{user `SshUsername`}}",
+      "tags": {
+        "BuiltBy": "Packer",
+        "Department": "{{user `Department`}}",
+        "Name": "{{user `ImageName`}}",
+        "OwnerEmail": "{{user `OwnerEmail`}}",
+        "Project": "{{user `Project`}}",
+        "SourceAmiId": "{{ .SourceAMI }}",
+        "SourceAmiName": "{{ .SourceAMIName }}"
+      },
+      "type": "amazon-ebs"
+    }
+  ],
+  "post-processors": [
+    {
+      "output": "manifest.json",
+      "strip_path": true,
+      "type": "manifest"
+    }
+  ],
+  "provisioners": [
+    {
+      "ansible_env_vars": [
```

Comment:
your missing the ImageName.  it needs to look like this.. https://github.com/Sage-Bionetworks/packer-base-ubuntu-bionic/blob/master/src/template.json#L42

Reply:
@zaro0508 Do I need that? It does not appear that way in the [packer-rstudio template](https://github.com/Sage-Bionetworks/packer-rstudio/blob/master/src/template.json#L42).
I was able to do a manual build without that.

Reply:
no, i finally figured it out that we pass it in with a packer -var flag